File: app/src/main/java/com/qinshou/rcvbaseadapterdemo/listener/OnItemClickListener.java
package com.qinshou.rcvbaseadapterdemo.listener;


import com.qinshou.rcvbaseadapterdemo.holder.BaseViewHolder;

/**
 * Description:
 * Created by 禽兽先生
 * Created on 2018/3/8
 */

public interface OnItemClickListener<T> {
    void onItemClick(BaseViewHolder holder, T t, int position);
}

File: rcvbaseadapter/src/main/java/com/qinshou/rcv/listener/OnItemClickListener.kt
package com.qinshou.rcv.listener

import com.qinshou.rcv.viewholder.BaseViewHolder


/**
 * Author: MrQinshou
 * Email:cqflqinhao@126.com
 * Date: 2021/4/14 11:17
 * Description:The listener of item click.
 */
interface OnItemClickListener<T> {
    fun onClick(holder: BaseViewHolder?, itemData: T?, position: Int)
}